src/apps/common/submodules/navigationWizard/navigationWizard.js

```diff
@@ -211,6 +211,16 @@ define(function(require) {
 					.on('click', function(event) {
 						event.preventDefault();
 
+						$this = $(this);
+
+						if ($this.hasClass('disabled')) {
+							return;
+						}
```

Comment:
How about modifying your target selector to NOT match elements with the `disabled` class instead?
```js
.find('selector:not(.disabled)')
```